inlong-sort/sort-connectors/mongodb-cdc/src/main/java/org/apache/inlong/sort/cdc/mongodb/DebeziumSourceFunction.java

```diff
@@ -505,11 +505,11 @@ public void deserialize(SourceRecord record, Collector<T> out) throws Exception
                                         source = value.getStruct(RecordUtils.DOCUMENT_TO_FIELD);
                                     }
                                     String dbName = source.getString(MongoDBEnvelope.NAMESPACE_DATABASE_FIELD);
-                                    String tableName = source.getString(MongoDBEnvelope.NAMESPACE_COLLECTION_FIELD);
+                                    String collectionName = source.getString(MongoDBEnvelope.NAMESPACE_COLLECTION_FIELD);
                                     SnapshotRecord snapshotRecord = SnapshotRecord.fromSource(source);
                                     boolean isSnapshotRecord = (SnapshotRecord.TRUE == snapshotRecord);
                                     sourceMetricData
-                                            .outputMetricsWithEstimate(dbName, tableName, isSnapshotRecord, value);
+                                            .outputMetricsWithEstimate(dbName, collectionName, isSnapshotRecord, value);
```

Comment:
mongo table level metric use collection. Not to use table. Full metric name is flink_taskmanager_job_task_operator_groupId_streamId_nodeId_database_collection_numRecordsIn.
You need modify this method `outputMetricsWithEstimate` that produce metric group and label.

Reply:
@chestnut-c please review here

Reply:
@kuansix You can pull code and modfiy here after the [PR](https://github.com/apache/inlong/pull/7007) is merge

Reply:
rebase it .